src/hybridmap/Mapping.cpp

```diff
@@ -0,0 +1,30 @@
+//
+// This file is part of the MQT QMAP library released under the MIT license.
+// See README.md or go to https://github.com/cda-tum/qmap for more information.
+//
+
+#include "hybridmap/Mapping.hpp"
+
+#include "hybridmap/NeutralAtomDefinitions.hpp"
+
+#include <stdexcept>
+
+namespace qc {
+void Mapping::applySwap(Swap swap) {
+  auto q1 = swap.first;
+  auto q2 = swap.second;
```

Comment:
Yes, I know this "trick". But for some reasons this gives me problems with Clion debugging, as it can not resolve the second version. So probably you can find multiple such things in my code, same for for loops. Not sure if this problem only exists on my end.

Reply:
To be fair, I never really had problems with that but I could see why structured bindings (that's the name for the "trick" ;o) ) could become confusing for debuggers.
In the end it's mostly syntactic sugar and creates easier to read code (typically), but it does not affect performance. So feel free to just leave it as-is.

Reply:
Ok, then I'll leave as it is, although I like the version with structured bindings more, but it can be really annoying during debugging.

Reply:
Maybe worth to investigate your debugging setup at some point. It should definitely be possible to get that working.
Quite naturally, debugging should happen on a `Debug` build. Maybe it's one of the compiler flags we enable that prevents a good debugging experience. Something to look into. Debuggability should not be the limiting factor for writing nice code.
Feel free to resolve this for now.